public/assets/js/scripts.js

```diff
@@ -27,6 +27,32 @@ var initMap = () => {
 $(document).ready(function () {
   $('#payment, #profile, #security').hide();
 
+  // initialize popovers
+  $(function () {
+    $('[data-toggle="popover"]').popover()
+  });
+
+  // check if browser is mobile to position popover
```

Comment:
This comment is not needed. It is best practice to define the width of the mobile as a constant to make the code more readable instead of the comment.

```js
const MOBILE_WIDTH = 576

...

if ($(window).width() < MOBILE_WIDTH) {
```